PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -567,6 +580,29 @@ void SetupContext(PostProcessRenderContext context)
             context.antialiasing = antialiasingMode;
             context.temporalAntialiasing = temporalAntialiasing;
             context.logHistogram = m_LogHistogram;
+
+#if UNITY_2018_2_OR_NEWER
+            context.physicalCamera = context.camera.usePhysicalProperties;
+            
+            LookAtConstraint lookAtConstraint = context.camera.gameObject.GetComponent<LookAtConstraint>();
+            if (lookAtConstraint && lookAtConstraint.sourceCount>0 && lookAtConstraint.enabled && lookAtConstraint.constraintActive)
+            {
+                List<ConstraintSource> sources = new List<ConstraintSource>();
```

Comment:
`new List<ConstraintSource>()` on every frame = GC pressure -> cache the list & `Clear()` it when needed.